src/tiler/src/util/layer-filter.js

```diff
@@ -19,6 +21,14 @@ const parsePromise = xmlString => new Promise((resolve, reject) => {
  * Set the status of each layer NOT in the list to false to disable
  */
 const filter = (xmlJson, enabledLayers) => {
+    const dne = enabledLayers.filter(a =>
+        xmlJson.Map.Layer.filter(b =>
+            b.$.name === a).length === 0)
```

Comment:
The two closing parens on the last line are odd at first glance and a little tricky to track down. I wonder if removing the last line break (i.e. putting the whole expression and the closing paren of the the `a` filter function on one line) would be an improvement.

Reply:
That seems to play ball with ESLint at least (which is why I ended up formatting it like this). At any rate, this is completely replaced in the filtering PR.